frontend/javascripts/oxalis/view/left-border-tabs/modals/add_volume_layer_modal.tsx

```diff
@@ -25,31 +31,48 @@ export default function AddVolumeLayerModal({
   >(null);
   const [newLayerName, setNewLayerName] = useState("Volume");
 
+  const datasetResolutionInfo = getDatasetResolutionInfo(dataset);
+  const [resolutionIndices, setResolutionIndices] = useState([0, 10000]);
+
   const handleSetNewLayerName = (evt: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) =>
     setNewLayerName(evt.target.value);
 
   const segmentationLayers = getSegmentationLayers(dataset);
   const volumeTracingLayers = getVolumeTracingLayers(dataset);
 
   const availableSegmentationLayers = _.differenceWith(segmentationLayers, volumeTracingLayers);
-
+  let selectedSegmentationLayer = null;
   const handleAddVolumeLayer = async () => {
     await api.tracing.save();
 
+    const minResolutionAllowed = Math.max(
+      ...datasetResolutionInfo.getResolutionByIndexOrThrow(resolutionIndices[0]),
+    );
+    const maxResolutionAllowed = Math.max(
+      ...datasetResolutionInfo.getResolutionByIndexOrThrow(resolutionIndices[1]),
```

Comment:
`resolutionIndices` are initialized to dummy values which would cause an error here if the user doesn't change the slider. I'll add another comment with a code snippet how to fix this.